app/assets/javascripts/discourse/app/components/modal/revise-and-reject-post-reviewable.js

```diff
@@ -0,0 +1,52 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { action } from "@ember/object";
+import { inject as service } from "@ember/service";
+import { isEmpty } from "@ember/utils";
+
+const OTHER_REASON = "Other...";
+
+export default class ReviseAndRejectPostReviewable extends Component {
+  @service siteSettings;
+
+  @tracked reason = null;
+  @tracked customReason = null;
+  @tracked feedback = null;
+  @tracked submitting = false;
+
+  get configuredReasons() {
+    return this.siteSettings.reviewable_revision_reasons
+      .split("|")
+      .concat([OTHER_REASON]);
+  }
```

Comment:
```suggestion
  get configuredReasons() {
    return this.siteSettings.reviewable_revision_reasons
      .split("|")
      .filter(Boolean)
      .concat([OTHER_REASON]);
  }
```

```javascript
"".split("|").concat(["no lavomatic"]) // ['', 'no lavomatic']
```

Reply:
I appreciate the laundromat jokes in review :laughing: Thanks :+1: